tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -759,6 +759,13 @@ error: Unknown parser error
    |                                               ^^^^ Unknown parser error
 
 
+error: Unknown parser error
```

Comment:
This seems like a bug in the parser... @MarkMcCulloh ?

Reply:
@eladb I am troubleshooting it locally and it's for sure a bug. I'm trying out different combinations and am baffled by how some of them passes but not the others.

Reply:
The message is bad but the syntax is indeed invalid, `Set{"c"}` should be `Set<str>{"c"}` (or just `{"c"}`)